mmhuman3d/core/conventions/keypoints_mapping/coco_wholebody.py

```diff
@@ -22,23 +22,41 @@
     'right_bigtoe',
     'right_smalltoe',
     'right_heel',
-    'face_contour_1',
-    'face_contour_2',
-    'face_contour_3',
-    'face_contour_4',
-    'face_contour_5',
-    'face_contour_6',
-    'face_contour_7',
-    'face_contour_8',
-    'face_contour_9',
-    'face_contour_10',
-    'face_contour_11',
-    'face_contour_12',
-    'face_contour_13',
-    'face_contour_14',
-    'face_contour_15',
-    'face_contour_16',
-    'face_contour_17',
+    # 'face_contour_1',
+    # 'face_contour_2',
+    # 'face_contour_3',
+    # 'face_contour_4',
+    # 'face_contour_5',
+    # 'face_contour_6',
+    # 'face_contour_7',
+    # 'face_contour_8',
+    # 'face_contour_9',
+    # 'face_contour_10',
+    # 'face_contour_11',
+    # 'face_contour_12',
+    # 'face_contour_13',
+    # 'face_contour_14',
+    # 'face_contour_15',
+    # 'face_contour_16',
+    # 'face_contour_17',
```

Comment:
Is the commented out keypoints is changed to the new ones or they are still necessary in some cases? It is quite confuse to keep them.

Reply:
Keypoints convention updated.